### PR TITLE
Node 18 support

### PR DESCRIPTION
**Pull request by piwysocki**

### CODE REVIEW

**Comment by ivanrosvimi** [comment]
Up!

**Comment by MiguelPraw** [comment]
UP

File: api/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
   "engineStrict": true,
   "engines": {
-    "node": "16.x"
```
Comment (dubzzz) [inline api/package.json:19]: Why not: >=16.x

Changing to 18.x would make users of node 16 not being able to import the library anymore. >=16.x seems to be a better compromise.

**Comment by plouc** [comment]
Node 18 is now supported (and used), fixed in https://github.com/plouc/nivo/pull/2313.

**Pull request closed without merging**